website/server/libs/user/validation.js

```diff
@@ -1,20 +1,38 @@
 import bannedSlurs from '../bannedSlurs';
+import bannedWords from '../bannedWords';
 import { getMatchesByWordArray } from '../stringUtils';
 import forbiddenUsernames from '../forbiddenUsernames';
 
 const bannedSlurRegexs = bannedSlurs.map(word => new RegExp(`.*${word}.*`, 'i'));
+const bannedWordsRegexs = bannedWords.map(word => new RegExp(`.*${word}.*`, 'i'));
 
-export function nameContainsSlur (username) {
-  for (let i = 0; i < bannedSlurRegexs.length; i += 1) {
-    const regEx = bannedSlurRegexs[i];
+export function stringContainsSlur (username) {
```

Comment:
the parameter here (and also in stringContainsBannedWord()) should be changed from `username` to something more generic